pulseapi/settings.py

```diff
@@ -25,6 +25,8 @@
     DEBUG=(bool, False),
     USE_S3=(bool, False),
     SSL_PROTECTION=(bool, False),
+    CORS_REGEX_WHITELIST=(tuple, ()),
```

Comment:
I believe that if you specify this as a tuple, comma delimited strings are automatically converted to the right format, so you shouldn't need to call `split(',')` on the variable below (if you use `env()` not `os.getenv()`)

Reply:
Ok, I was just copying from the network site settings 😁